dhash/database.py

```diff
@@ -102,6 +103,59 @@ def dump(self) -> dict:
         }
 
 
+class HashConfig(database.base):
+    """Stores config in format key:value"""
```

Comment:
Please add a note that this is configured per-bot and not per-guild, and why. We may not remember that a year from now.

Reply:
Done.